wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/listener/BluetoothReceiver.java

```
@@ -930,7 +930,11 @@ public void onScanFailed(int errorCode) {
                         }
                     }
                     break;
-                default:
```

Probably should still have a default case, maybe just to log so not silently failing?

---

yeah, it warns, but I'll re-add it (basically the lint error was "you're not addressing all these cases specifically")